uberfire-project/uberfire-project-client/src/main/java/org/guvnor/common/services/project/client/security/ProjectController.java

```diff
@@ -41,7 +41,12 @@
 import org.jboss.errai.common.client.api.Caller;
 import org.jboss.errai.security.shared.api.identity.User;
 import org.uberfire.client.promise.Promises;
+import org.uberfire.rpc.SessionInfo;
+import org.uberfire.security.ResourceRef;
 import org.uberfire.security.authz.AuthorizationManager;
+import org.uberfire.workbench.model.ActivityResourceType;
+
+import static org.kie.workbench.common.workbench.client.PerspectiveIds.SERVER_MANAGEMENT;
```

Comment:
I missed this in my previous review, you can't depend on kie-wb-common in appformer. In this case, this new method should be somewhere in kie-wb-common, because the perspective is from there.

Reply:
thank you!
fixed